neca/statics/lib/core.js

```diff
@@ -1,12 +1,15 @@
 // connect a websocket to the server
 
-const SERVER_URL = "localhost:8080"
-var socket = io();
-
-socket.on('connect', function() {
-    console.log("Connected to server");
+const socket = io({
+    autoConnect: false
 });
 
+document.addEventListener("DOMContentLoaded", () => {
+    socket.connect();
+    socket.on('connect', function () {
+        console.log("Connected to server");
+    });
```

Comment:
you can leave this piece out of the event handler (and put it where it was before)